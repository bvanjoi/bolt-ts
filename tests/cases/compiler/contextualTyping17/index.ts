var foo: {id:number;} = {id:4}; foo = {id: 5, name:"foo"};
//~^ ERROR: Object literal may only specify known properties, and 'name' does not exist.