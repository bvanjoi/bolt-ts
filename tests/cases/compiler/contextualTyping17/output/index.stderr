  × Object literal may only specify known properties, and 'name' does not
  │ exist.
   ╭─[../../tests/cases/compiler/contextualTyping17/index.ts:3:47]
 3 │ name:"foo"
   · ──────────
   ╰────
